### PR TITLE
feat: Add TLS cert lifetime setter to SecretOperatorVolumeSourceBuilder

### PR DESCRIPTION
**Pull request by razvan**
# Description

Part of: https://github.com/stackabletech/issues/issues/586

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

**Review by sbernauer — APPROVED** [review]
Change LGTM. Can we please rename the PR to something like `feat: Add TLS cert lifetime setter to SecretOperatorVolumeSourceBuilder`

File: crates/stackable-operator/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
-All notable changes to this project will be documented in this file.
-
 ## [Unreleased]
 
+### Added
+
+- Added cert lifetime setter to `SecretOperatorVolumeSourceBuilder` ([#915])
+
+All notable changes to this project will be documented in this file.
```
Comment (NickLarsenNZ) [inline crates/stackable-operator/CHANGELOG.md:9]: I think this line got accidentally moved here. It should be reverted.

**Review by NickLarsenNZ — CHANGES_REQUESTED** [review]
LGTM, just need a fixed up changelog

File: crates/stackable-operator/CHANGELOG.md (hunk 1 above)
Comment (razvan) [reply crates/stackable-operator/CHANGELOG.md]: Oops! Fixed.

File: crates/stackable-operator/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
 # Changelog
 
+## [Unreleased]
```
Comment (sbernauer) [inline crates/stackable-operator/CHANGELOG.md:3]: Please have another look at the diff, there is still something unwanted going on here ;)

Comment (razvan) [reply crates/stackable-operator/CHANGELOG.md]: dammit! [fixed](https://github.com/stackabletech/operator-rs/pull/915/commits/e2f7f7f144229f5f50230343ee0157f290788dee)

**Review by sbernauer — APPROVED** [review]
Thanks!

**Review by sbernauer — APPROVED** [review]

**Review by sbernauer — APPROVED** [review]

**Pull request merged**